apps/coscrad-frontend/src/components/links/links.tsx

```diff
@@ -0,0 +1,36 @@
+import LaunchIcon from '@mui/icons-material/Launch';
+import { Card, Link, Typography } from '@mui/material';
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { ExternalLink } from '../../configurable-front-matter/data/configurable-content-schema';
+import { FunctionalComponent } from '../../utils/types/functional-component';
+
+const LinkPresenter = ({ title, url, description }: ExternalLink) => {
+    return (
+        <Card sx={{ p: 2, mb: 2 }}>
+            <Typography mb={0.5} variant="h3">
+                {title}
+            </Typography>
+            <Typography variant="body1" color="text.secondary">
```

Comment:
Two questions: should we have both the URL and a "Visit" link?  or just the hyperlinked URL?  Also, it might be worth truncating the URL in case it's long and doesn't wrap on mobile or other small viewports.  There's apps/coscrad-frontend/src/components/resources/utils/truncate-text-for-display.tsx

Reply:
No, we don't want to show the URL. They can always "copy link" if they want to see it. The point of adding the title is that we can use the title as the text for the link. I'll have to check if this change has been made yet.

Reply:
BTW I'm removing the URL for now. If Justin and Geoff decide it is indeed important to show the user, we can add this back later.